subprojects/gradle/design-screenshots/src/main/kotlin/com/avito/plugin/ScreenshotsPlugin.kt

```diff
@@ -10,15 +10,17 @@ class ScreenshotsPlugin : Plugin<Project> {
     override fun apply(project: Project) {
         project.withAndroidApp { appExtension ->
             appExtension.applicationVariants.all { applicationVariant ->
-                project.tasks.register<PullScreenshotsTask>("recordScreenshots") {
-                    group = "design"
-                    variant.set(applicationVariant)
-                    description = "Create and pull screenshots from device"
-                }
-                project.tasks.register<ClearScreenshotsTask>("clearScreenshots") {
-                    group = "design"
-                    variant.set(applicationVariant)
-                    description = "Clear screenshots on device"
+                if (applicationVariant.name != "release") {
```

Comment:
Why do we skip the release application variant?

Reply:
this plugin was written without variants in mind, it fails on basic integration test because trying to register task with same name for every variant;
instead fixing it proper way (i don't know all applications so post poned that) i fixed it the way it's actually used (run manually for debug or stagingAutoTest) variant

Reply:
@JaverSingleton , cc